patches/api/0300-Missing-Entity-API.patch

```diff
@@ -1128,6 +1128,31 @@ index a52a7af219633d575dcbe8ac4b219834bfd4d4d2..1e839b247182af6873a4d74b236d6412
   */
 -public interface Salmon extends Fish { }
 +public interface Salmon extends io.papermc.paper.entity.SchoolableFish { } // Paper - Schooling Fish API
+diff --git a/src/main/java/org/bukkit/entity/TNTPrimed.java b/src/main/java/org/bukkit/entity/TNTPrimed.java
+index 0813bd913c8fdb2001963ce3e82c07c2af105418..78ee6fa4f0e420fbbbc4d236c0b2e691b8ed7c94 100644
+--- a/src/main/java/org/bukkit/entity/TNTPrimed.java
++++ b/src/main/java/org/bukkit/entity/TNTPrimed.java
+@@ -64,4 +64,20 @@ public interface TNTPrimed extends Explosive {
+     default org.bukkit.Location getSourceLoc() {
+         return this.getOrigin();
+     }
++    // Paper start
++    /**
++     * Gets the data of the block shown by this TNT entity.
++     *
++     * @return BlockData of the shown block
++     */
++    @org.jetbrains.annotations.NotNull
++    org.bukkit.block.data.BlockData getBlock();
```

Comment:
Might need better naming.

tnt.getBlock() I'd expect to return a block somewhere, idk maybe the block the tnt originated ?
This is more like the displayed block data ?